src/main/java/org/terasology/structureTemplates/internal/events/SendRegionEvent.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.structureTemplates.internal.events;
+
+import org.terasology.engine.entitySystem.event.Event;
+import org.terasology.structureTemplates.components.SpawnBlockRegionsComponent;
+
+import java.util.List;
+
+public class SendRegionEvent implements Event {
```

Comment:
This event needs documentation.

Where is the region sent to? Who should react on this event? What kind of event is this?